godel_process_path_generation/src/process_path_generator_node.cpp

```diff
@@ -26,7 +26,7 @@
 #include <boost/tuple/tuple.hpp>
 #include <godel_process_path_generation/utils.h>
 #include <godel_process_path_generation/VisualizeBlendingPlan.h>
-#include <godel_msgs/ProcessPlanning.h>
+#include <godel_msgs/ProcessPlanningAction.h>
```

Comment:
Process Planning is now an action as opposed to a service. Converted the header.